gpkit/tests/t_small.py

```diff
@@ -45,8 +47,19 @@ def test_mul_add(self):
         self.assertEqual(a * b * c, HashVector())
         self.assertEqual(a * {'x': 6, 'k': 4}, HashVector(x=6))
 
+class TestResultsTable(unittest.TestCase):
+    """TestCase for results_table()"""
+
+    def test_nan_printing(self):
+        """Test that solution prints when it contains nans"""
+        x = VarKey(name='x')
+        data = {x: np.array([np.nan,   1.,   1.,   1.,   1.])}
+        title = "Free variables"
+        printstr = results_table(data, title)
+        targetstr = "Free variables\n--------------\nx : [ nan       1         1         1        ... ] \n"
+        self.assertEqual(printstr, targetstr)
```

Comment:
This looks fine. Might want to consider a slightly less strict test, so that this won't immediately fail if we change the format of `results_table`. It's not a big deal to change the unit test too in that case, but better might be something like

`self.assertTrue("nan" in printstr)`
`self.assertTrue(title in printstr)`